cpu/stm32_common/periph/flashpage.c

```diff
@@ -27,12 +27,33 @@
 
 #include "periph/flashpage.h"
 
+#if defined(CPU_FAM_STM32L0)
+#define CNTRL_REG              (FLASH->PECR)
+#define CNTRL_REG_LOCK         (FLASH_PECR_PELOCK)
+#define KEY_REG                (FLASH->PEKEYR)
+#define FLASH_CR_PER           (FLASH_PECR_ERASE | FLASH_PECR_PROG)
+#define FLASH_CR_PG            (FLASH_PECR_FPRG | FLASH_PECR_PROG)
+#define FLASHPAGE_LIMIT        (FLASHPAGE_SIZE / 4)
+#else
+#define CNTRL_REG              (FLASH->CR)
+#define CNTRL_REG_LOCK         (FLASH_CR_LOCK)
+#define KEY_REG                (FLASH->KEYR)
+#define FLASHPAGE_LIMIT        (FLASHPAGE_SIZE / 2)
```

Comment:
This `FLASHPAGE_LIMIT` define is not documented and thus I don't see what is really about.